packages/calling-component-bindings/src/callUtils.ts

```diff
@@ -5,13 +5,21 @@ import { DeviceManagerState, StatefulCallClient } from '@internal/calling-statef
 import { CallState as CallStatus } from '@azure/communication-calling';
 
 /**
- * Check if the call state represents being connected to a call
+ * Check if the call state represents being in the call
  *
  * @internal
  */
 export const _isInCall = (callStatus?: CallStatus): boolean =>
   !!callStatus && !['None', 'Disconnected', 'Connecting'].includes(callStatus);
```

Comment:
why keeping the "Connecting" state here if it's in "_isInLobbyOrConnecting", unless it's used elsewhere and need this state

Reply:
`_isInCall` is being used in many places. Can't touch it without breaking stuff...